### DISCUSSION TITLE
rs-232 connections

### DISCUSSION THREAD

**Discussion by kaoplx** [netbox-community/netbox#13260] [Q&A]
I have a device that has 2 Ethernet interfaces and 5 de-9. behind each db9 port, I have several devices (up to 10 pieces(bus)) that exchange over rs-232 with my device. how do I display these connections using a netbox?
I tried to use a patch panel, but it has a one-to-one relationship between the front and rear ports. And I need that one front port could correspond to several rear ones. I know that this is in version 3.3, but now the current 3.5.6.
![IMG_2023_07_25_14_28_49_2305006580701478220913](https://github.com/netbox-community/netbox/assets/71017633/9e02896e-7b74-4c54-95a1-7b2ebf472b15)

**Comment by candlerb** [comment]
Netbox 3.3 and later support cables with multiple terminations at each end.  So you should be able to:

* Create a "console server port" called COM3 on the server
* Create a "console port" on each of dev1, dev2, dev3
* Create a single cable with a connection to server COM3 at one end, and connections to dev1 console, dev2 console, dev3 console at the other end

The user interface for doing this in Netbox 3.5+ is harder to use than 3.3/3.4, and I haven't tested it with console connections explicitly, but I don't see why it shouldn't work.

(Aside: I didn't think RS232 supports bus operations; I thought you needed RS485 for that?)

**Reply by candlerb** [reply]
> I need that one front port could correspond to several rear ones. I know that this is in version 3.3, but now the current 3.5.6.

Multiple front ports can link to a single rear port (it's always been that way round), but this is for when you're multiplexing connections, e.g. multiple LC connectors onto an MPO connector. It doesn't support sharing the connection, as for example you'd want to do with a GPON splitter, which would be the optical equivalent of the "bus" you are building.
